<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkersForm</class>
 <widget class="QWidget" name="WorkersForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>682</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>371</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Доглядачі</string>
   </property>
   <widget class="QTableView" name="caretakersTableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>351</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>781</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Внести нового співробітника</string>
   </property>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Тип співробітника</string>
    </property>
    <widget class="QCheckBox" name="caretakerCheckBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>40</y>
       <width>78</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Доглядач</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="veterinariancheckBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>90</y>
       <width>78</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Ветеринар</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>150</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Внести дані</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="insertGroupBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>30</y>
      <width>431</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Заповнення</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Прізвище та ініціали</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Номер телефону</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Дата народження</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="pibText">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="phoneNumText">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextEdit" name="dateOfBirthText">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>100</y>
       <width>271</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>781</width>
     <height>201</height>
    </rect>
   </property>
   <property name="title">
    <string>Вибрати</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>                        ПІБ</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="pibText_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>371</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string>Доглядач/Співробітник</string>
    </property>
    <widget class="QTableView" name="employeersViewer">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>351</width>
       <height>101</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="updateButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>50</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Редагувати</string>
    </property>
   </widget>
   <widget class="QPushButton" name="searchButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Знайти</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="caretakerCheckBox_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Доглядач</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="veterinariancheckBox_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>78</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Ветеринар</string>
    </property>
   </widget>
   <widget class="QPushButton" name="updateButton_2">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>90</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Переглянути тварин</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>40</y>
     <width>371</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Ветеринари</string>
   </property>
   <widget class="QTableView" name="veterinariansTableView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>351</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
